demo/Demo/Buttons.elm

```diff
@@ -214,22 +214,74 @@ viewButtons model =
 
 view : Model -> Html Msg
 view model = 
-  Page.body2 "Buttons" srcUrl intro references  
+  Page.body1' "Buttons" srcUrl intro references
     [ p [] 
         [ text """Various combinations of colors and button styles can be seen
                   below. Most buttons have animations; try clicking. Code for the
                   last clicked button appears below the buttons."""
         ]
     , Grid.grid [] (viewButtons model)
-    , p []
+    , Options.styled p [ Options.css "min-height" "250px" ]
         [ model.last 
            |> Maybe.map describe 
            |> Maybe.map (\str -> "Code for '" ++ str ++ "':")
            |> Maybe.withDefault "Click a button to see the corresponding code."
            |> text
         , Code.view model.code [ Options.css "margin" "20px" ]
         ]
-    ] 
+    ]
+    [ h4 [ id "link-buttons" ]
+        [ text "Link buttons"
+        ]
+
+    , p []
+        [ text """Link buttons are links that look and act like buttons.
+                However, they also support attributes for links such as 'href' and 'target'."""
```

Comment:
You are correct! `Button.link` only supports the attributes that have been specified as LinkProperties